core/src/components/tabs/test/standalone/e2e.ts

```diff
@@ -0,0 +1,26 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+test('tabs: standalone', async () => {
+  const page = await newE2EPage({
+    url: '/src/components/tabs/test/standalone?ionic:_testing=true'
+  });
+
+  const compares = [];
+
+  // Initial page load
+  compares.push(await page.compareScreenshot());
+
+  let tabButton = await page.find('#tab-button-tab-one');
+  await tabButton.click();
+
+  compares.push(await page.compareScreenshot(`tab one`));
+
+  tabButton = await page.find('#tab-button-tab-two');
+  await tabButton.click();
```

Comment:
Do we need to wait for the tab content to become visible? just worried about a slow CI running these

Reply:
I basically just moved the vanilla tabs test to be standalone. This is how it works in the basic test so it should work the same here.